db/migrate/20220722064646_add_requested_flowcell_type_to_request_metadata.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Adding requested_flowcell_type column to request_metadata
+class AddRequestedFlowcellTypeToRequestMetadata < ActiveRecord::Migration[6.0]
+  def change
+    add_column :request_metadata, :requested_flowcell_type, :string
+  end
+end
```

Comment:
This should be an association with the flowcell_types table, not a plain string.